monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -196,13 +202,130 @@ Appears in: link:#clustermonitoringconfiguration[ClusterMonitoringConfiguration]
+
+|netdev|link:#nodeexportercollectornetdevconfig[NodeExporterCollectorNetDevConfig]|Defines the configuration of the `netdev` collector, which collects network devices statistics. Enabled by default.
+
+|netclass|link:#nodeexportercollectornetclassconfig[NodeExporterCollectorNetClassConfig]|Defines the configuration of the `netclass` collector, which collects information about network devices. Enabled by default.
+
+|buddyinfo|link:#nodeexportercollectorbuddyinfoconfig[NodeExporterCollectorBuddyInfoConfig]|Defines the configuration of the `buddyinfo` collector, which collects statistics about memory fragmentation from the `node_buddyinfo_blocks` metric. This metric collects data from `/proc/buddyinfo`. Disabled by default.
+
+|===
+
+== NodeExporterCollectorCpufreqConfig
+
+=== Description
+
+The `NodeExporterCollectorCpufreqConfig` resource works as an on/off switch for the `cpufreq` collector of the `node-exporter` agent. By default, the `cpufreq` collector is disabled. Under certain circumstances, enabling the cpufreq collector increases CPU usage on machines with many cores. If you enable this collector and have machines with many cores, monitor your systems closely for excessive CPU usage.
+
+Appears in: link:#nodeexportercollectorconfig[NodeExporterCollectorConfig]
+
+[options="header"]
+|===
+| Property | Type | Description
+|enabled|bool|A Boolean flag that enables or disables the `cpufreq` collector.
+
+|===
+
+== NodeExporterCollectorNetClassConfig
+
+=== Description
+
+The `NodeExporterCollectorNetClassConfig` resource works as an on/off switch for the `netclass` collector of the `node-exporter` agent. By default, the `netclass` collector is enabled. If disabled, these metrics become unavailable: `node_network_info`, `node_network_address_assign_type`, `node_network_carrier`, `node_network_carrier_changes_total`, `node_network_carrier_up_changes_total`, `node_network_carrier_down_changes_total`, `node_network_device_id`, `node_network_dormant`, `node_network_flags`, `node_network_iface_id`, `node_network_iface_link`, `node_network_iface_link_mode`, `node_network_mtu_bytes`, `node_network_name_assign_type`, `node_network_net_dev_group`, `node_network_speed_bytes`, `node_network_transmit_queue_length`, `node_network_protocol_type`.
+
+Appears in: link:#nodeexportercollectorconfig[NodeExporterCollectorConfig]
+
+[options="header"]
+|===
+| Property | Type | Description
+|enabled|bool|A Boolean flag that enables or disables the `netclass` collector.
+
+|useNetlink|bool|A Boolean flag that activates the `netlink` implementation of the `netclass` collector. This implementation improves the performance of the `netclass` collector by omitting these metrics: `node_network_address_assign_type`, `node_network_name_assign_type`, `node_network_device_id`, `node_network_speed_bytes`. In addition, The `node_network_info` metric lacks the `duplex` label.
```

Comment:
Do we need to hint that `netlink` is disabled by default? Although it's currently fixed in 4.14, it should be backported to 4.13 soon.
see from https://issues.redhat.com/browse/OCPBUGS-8282

Reply:
@raptorsun ^